yarn.lock

```diff
@@ -16,43 +16,46 @@
     esutils "^2.0.2"
     js-tokens "^4.0.0"
 
-acorn-jsx@^4.1.1:
-  version "4.1.1"
-  resolved "https://registry.yarnpkg.com/acorn-jsx/-/acorn-jsx-4.1.1.tgz#e8e41e48ea2fe0c896740610ab6a4ffd8add225e"
-  dependencies:
-    acorn "^5.0.3"
+acorn-jsx@^5.0.0:
+  version "5.0.1"
+  resolved "https://repository.dmz.croz.net/repository/npm-all/acorn-jsx/-/acorn-jsx-5.0.1.tgz#32a064fd925429216a09b141102bfdd185fae40e"
```

Comment:
@mfolnovic This doesn't look right. It's not cool to commit these things to the repo, especially as they're really hard to detect.